src/testHelpers/JsonActionReader.ts

```diff
@@ -10,7 +10,7 @@ export class JsonActionReader extends AbstractActionReader {
     protected onlyIrreversible: boolean = false,
     protected maxHistoryLength: number = 600,
```

Comment:
`maxHistoryLength` no longer exists.

Reply:
Looks like its still on the ActionReaderOptions https://github.com/EOSIO/demux-js/blob/develop/src/interfaces.ts#L19

Reply:
Oh, well would you look at that. Haha, that needs to be removed, I'll make a really quick PR for it after this is merged.

Reply:
I can remove it, then update this PR to match it. So that we don't have to come back to this code and update it again